www/assets/app.css

```diff
@@ -29,214 +24,240 @@ a:hover {
   text-decoration: underline;
 }
 
-h1, h4 {
-  margin-top: 0;
-  margin-bottom: 0;
-}
-
-h1 {
-  line-height: 1.5;
-}
-
-textarea {
-  height: 3.25em;
-  max-width: 100%;
+label {
+  font-weight: bold;
+  margin-bottom: 8px;
 }
 
-/* Layout */
-
-.table td input {
-  width: 100%;
+button,
+input {
+  color: inherit;
+  font: inherit;
+  margin: 0;
 }
-
-.container {
-  margin-left: auto;
-  margin-right: auto;
-  padding-left: 1rem;
-  padding-right: 1rem;
-  max-width: 600px;
+button {
+  cursor: pointer;
 }
-
-/* Components */
-
-.signedin-message {
-  display: none;
-}
-
-[data-account-state='signed-out'] .account-login,
-[data-account-state='signed-out'] .password-reset {
-  display: block;
+button.small {
+  padding: .5rem;
+  font-size: .8rem;
 }
-
-[data-account-state='signed-in'] .account-settings {
+input {
   display: block;
+  padding: 1rem;
+  height: 50px;
+  width: 100%;
 }
 
-[data-account-state='signed-in'] .signedin-message {
-  display: inline-block;
-}
-
-[data-account-state='signed-in'] .account-login,
-[data-account-state='signed-in'] .login-signup-link {
-  display: none;
+table {
+  width: 100%;
 }
 
-.main-header {
+/* Header */
+body > header {
```

Comment:
Generally it's better to use classes like it was previously. Since it's lower specificity, with these changes the CSS is less flexible as it's tied closer to the HTML.

https://www.smashingmagazine.com/2007/07/css-specificity-things-you-should-know/